### ISSUE TITLE
Meta issue for structure and overall approach 

### ISSUE THREAD

**Issue by davanstrien** [carpentries-incubator/machine-learning-librarians-archivists#87]
Possible structure from discussion with @mark-bell-tna 

- Start with applications and uses of machine learning right at the start. What can machine learning do. 
	- Maybe start with generative AI? even how you might use for emails
	- Then more specific tasks: creating search queries i.e. using chat GPT to help create advanced queries
	- More GLAM specific examples (more about projects) 
- Then move to ethics and limitations
- Machine learning projects? 
- How does this work:
    - Impact of data?
    -

**Comment by davanstrien** [comment]
Other tasks for lesson
- template slides for delivery

**Comment by MikeTrizna** [comment]
Yes! I absolutely agree with getting right into "unstructured data" Tasks! Although do we have to caveat the title or the introduction? 

I wish there was a better, beginner-friendly term to differentiate between tabular structured data vs. unstructured text, images, audio, etc. When I brought up the term at the workshop last week, there was immediate pushback that their image of text datasets were organized and standardized, and thus structured.

**Comment by leighphan** [comment]
@davanstrien @mark-bell-tna I agree this is a great direction to updating the lesson structure. I included some feedback on the general structure of the lesson that can apply here as we consider restructuring in 
https://github.com/carpentries-incubator/machine-learning-librarians-archivists/issues/82#issuecomment-1683079126.

During the sprint I believe @mark-bell-tna mentioned an earlier discussion about including the environmental impacts of using generative AI in the lesson, which I think would make sense to include in the "How does this work: Impact of data?" section.
